_docs/2.0.0/usage/config.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: docs
+title: Usage - Configuration
+version: '2.0.0'
+categories:
+    - '2.0.0'
+    - usage
+redirect_from: /docs/usage/config
+---
+
+Configuration
+============
+
+<!-- toc -->
+- [Configuration](#configuration)
+  - [Intro](#intro) 
+  - [General options](#general-options)
+<!-- tocstop -->
+
+Intro
+-----------
+
+This page describes the usage of configuration of _Standardization_ and _Conformance_.
+There are number of default options that 
+[Project's readme](https://github.com/AbsaOSS/enceladus/blob/develop/README.md) documents
```

Comment:
I think it would be better to refer the `master`

Reply:
I have thought about it first, but then I saw https://github.com/AbsaOSS/enceladus/blame/gh-pages/_docs/2.0.0/usage/schema.md#L150 where `develop` is also referred to.

So, I'd better change that, too, I guess.

Reply:
Updated.

Reply:
Yeah, it was an oversight. Thanks for fixing